client/src/api/api_client.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+/**
+ * Client to interface with API.
+ */
+class ApiClient {
```

Comment:
is this .js file included intentionally? I don't think we typically push / see the js files which contain compiled ts code

Reply:
Removed from git